src/darsia/image/image.py

```diff
@@ -240,8 +240,11 @@ def __sub__(self, other: da.Image):
         """
         if self.img.shape != other.img.shape:
             warn("Images have different shapes. Resizing second argument to match.")
-            other_img = cv2.resize(other.img, self.img.shape[::-1][1:3])
-            return da.Image(self.img - other_img, copy.copy(self.metadata))
+            return da.Image(
+                self.img
+                - cv2.resize(np.copy(other.img), tuple(reversed(self.img.shape[:2]))),
```

Comment:
Not sure whether the copy is needed here. I do not see how the resize would affect the image. But it is OK for me.

Reply:
I don't know. I was just afraid that cv2's resize function modified the original image. But I checked now, and it doesnt. Ill remove the copy and merge then.